src/Xamarin.Android.Tools.AndroidSdk/AndroidAppManifest.cs

```diff
@@ -150,6 +150,11 @@ public void WriteToFile (string fileName)
 			set { application.SetAttributeValue (aNS + "icon", NullIfEmpty (value)); }
 		}
 
+		public string? ApplicationIconRound {
```

Comment:
If we follow the pattern of the other properties here, should this be named:
```suggestion
		public string? ApplicationRoundIcon {
```
This matches the XML-name more closely.